scripts/README.md

```diff
@@ -0,0 +1,16 @@
+##Scripts for chart verifier workflows
```

Comment:
Would be nice to add some top-level docstring to each python module: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I added info to each of the new python scripts created, but not those copied from the charts repo.